### ISSUE TITLE
Plz how to add Virtual loss 

### ISSUE THREAD

**Issue by johnpennyson** [binary-com/binary-bot#878]
Plz gurus in the house how do i add virtual loss to a bot script, that is I want a situation where the trade will count 3 loss before executing the real trade. How do i create the algorithm on binary bot. Thanks

**Comment by Baselope** [comment]
If I understand correctly - virtual loss = virtual account & real trade = real account.
- Not possible in the GUI - the account portion of block 1 is only loaded once at the start of the bot - never again.
- It can be done in the CLI where the account information is passed before every contract preparation. I.e. you can start the bot in VRTC, trade 3 times, switch to real account, trade once, switch to VRTC, etc. Do some reading in this forum for the required information, everything you need is here.

It is possible to program a dummy trade (NOT virtual) but you need good accuracy (don't use tick or even second trades - minute trades can work, the longer the better the accuracy) and a defined delay that mimics your trade - you will have to set the spot values yourself and evaluate the results inside the bot.

**Comment by binary-ex-machina** [comment]
virtual loss is a illusion. If you use fixed value for it, you are putting that the market will always have this number of losses, in order to avoid big losing streaks, which is not true.

**Comment by gokula-krishna-dev** [comment]
We're closing this issue due to inactivity, you're welcome to open it again if you still haven't got your answer.
